contracts/diamond/init/DiamondInit.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+// It is expected that this contract is customized if you want to deploy your diamond
+// with data from a deployment script. Use the init function to initialize state variables
+// of your diamond. Add parameters to the init funciton if you need to.
+
+// Adding parameters to the `init` or other functions you add here can make a single deployed
+// DiamondInit contract reusable accross upgrades, and can be used for multiple diamonds.
+
+import "../introspection/DiamondERC165.sol";
+
+contract DiamondInit is DiamondERC165 {
```

Comment:
I don't think this contract is actually needed. Why not have `__DiamondERC165_init()` function directly in the `DIamondERC165` contract?